candi/bashible/bundles/centos/bootstrap.sh.tpl

```diff
@@ -40,6 +40,13 @@ if [ "${VERSION_ID}" == "8" ] ; then
   alternatives --set python /usr/bin/python3
 fi
 
+for FS_NAME in `egrep -v -e '(^[[:space:]]*$|^#)' /etc/fstab | awk '{ print $1 }'`; do
+  if  [[ `xfs_info $FS_NAME | grep --count ftype=0` -eq 1 ]]; then
+     echo "XFS file system with fstop=0 was found on the node ($FS_NAME). This may cause problems (https://www.suse.com/support/kb/doc/?id=000020068), please fix it and try again."
```

Comment:
```suggestion
     >&2 echo "XFS file system with fstype=0 was found ($FS_NAME). This may cause problems (https://www.suse.com/support/kb/doc/?id=000020068), please fix it and try again."
```